connectors/src/resources/slack_configuration_resource.ts

```diff
@@ -111,14 +111,35 @@ export class SlackConfigurationResource extends BaseResource<SlackConfigurationM
     return new this(this.model, blob.get());
   }
 
-  async isBotWhitelistedToSummon(botName: string | string[]): Promise<boolean> {
-    return !!(await SlackBotWhitelistModel.findOne({
-      where: {
-        connectorId: this.connectorId,
-        botName: botName,
```

Comment:
I think you can change only  `botName: botName` to `...(botName ? { botName} : { botId })` here to not have write all the code below